background/background.go

```diff
@@ -38,9 +44,13 @@ func Start(processes Processes, args interface{}) *T {
 		finished := make(chan struct{})
 		register.s[i].shutdown = shutdown
 		register.s[i].finished = finished
+		register.Add(1)
 		go func(p Process, shutdown <-chan struct{}, finished chan<- struct{}) {
 			// pass the shutdown to the Run loop for shutdown signalling
-			p.Run(args, shutdown)
+			func() {
```

Comment:
What does having this func to enclose the run and done do? Is there something special or can the goroutine just do run+done+close?

Reply:
This is a wrapper function during design and is not in used anymore. I will remove it.